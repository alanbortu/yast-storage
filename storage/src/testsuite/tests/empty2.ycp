// testedfiles: helper1b.ycp
{
    include "./lib/setup-system.ycp";

    setup_system("empty");

    include "./lib/helper1a.ycp";

    import "ProductFeatures";

    ProductFeatures::SetBooleanFeature("partitioning", "try_separate_home", true);
    ProductFeatures::SetBooleanFeature("partitioning", "proposal_lvm", false);
    ProductFeatures::SetStringFeature("partitioning", "vm_desired_size", "30 GB");
    ProductFeatures::SetStringFeature("partitioning", "root_base_size", "20 GB");

    include "./lib/helper1b.ycp";
}
