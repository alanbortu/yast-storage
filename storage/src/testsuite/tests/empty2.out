Dump	Proposal:
Dump	Create swap partition /dev/sda1 (2.01 GB)
Dump	Create root partition /dev/sda2 (20.00 GB) with ext4
Dump	Create partition /dev/sda3 (276.08 GB) for /home with ext4
